config/api.yml

```diff
@@ -742,6 +742,25 @@
       :post:
       - :name: assign
       - :name: unassign
+  :firmwares:
+    :description: Firmwares
+    :options:
+    - :collection
+    - :subcollection
```

Comment:
@rodneyhbrown7 it looks like in the linked PR it's not declaring this as a subcollection. It may be best to leave this out in this PR, or until you are ready to add it to the phyical server config